assets/js/vue/components/data-land/DownloadZipBtn.vue

```diff
@@ -0,0 +1,154 @@
+                    </div>
+                    <hr>
+                    <div>
+                        <strong>UDI:</strong> {{ datasetInfo.dataset.udi }}<br>
+                        <strong>File name:</strong> {{ datasetInfo.dataset.filename }}<br>
+                        <strong>File size:</strong> {{ datasetInfo.dataset.fileSize }}<br>
+                        <strong>SHA256 Checksum:</strong> {{ datasetInfo.dataset.checksum }}<br>
+                        <strong>Estimated Download Time:</strong>{{ datasetInfo.dataset.fileSizeRaw }}<br>
+                    </div>
+                </div>
+                <DxToolbarItem v-if="!datasetInfo.remotelyHosted"
+                               widget="dxButton"
+                               toolbar="bottom"
+                               location="center"
+                               :options="downloadButtonOptions"
+                />
+            </template>
+        </DxPopup>
+    </div>
+</template>
+
+<script>
+import "devextreme/dist/css/dx.common.css";
+import "devextreme/dist/css/dx.light.css";
+import {DxPopup, DxPosition, DxToolbarItem} from "devextreme-vue/popup";
+import {getApi} from "@/vue/utils/axiosService";
+
+export default {
+    name: "DownloadZipBtn",
+    components: {
+        DxPopup,
+        DxPosition,
+        DxToolbarItem,
+    },
+    props: {
+        id: {
+            required: true
+        }
+    },
+    data() {
+        return {
+            showDownloadDialog: false,
+            datasetInfo: {
+                remotelyHosted: false,
+                dataset: {},
+                fileUri: ""
+            },
+            buttonTitle: "",
+            additionalInfo: "",
+            downloadButtonOptions: {
+                icon: 'download',
+                text: 'Download',
+                onClick: () => {
+                    const url = `${Routing.generate("pelagos_app_download_http", {"id": this.id})}`;
+                    const link = document.createElement('a');
+                    link.href = url;
+                    document.body.appendChild(link);
+                    setTimeout(function() {
+                        document.body.removeChild(link);
+                        window.URL.revokeObjectURL(url);
+                    }, 0);
+                    link.click();
+                    notify({
+                        message: message,
+                        position: {
+                            my: 'center top',
+                            at: 'center top'
+                        }
+                    }, 'success', 3000);
+                }
+            },
+        }
+    },
+    created() {
+        getApi(Routing.generate("pelagos_app_download_default", {"id": this.id})).then(response => {
+            this.datasetInfo = response.data;
+            if (this.datasetInfo.remotelyHosted) {
+                this.buttonTitle = "External Dataset Link"
+                if (this.datasetInfo.dataset.availability === 5) {
```

Comment:
I don't think this state is ever reachable, since we don't display the download link for restricted datasets, regardless of whether or not it's remotely hosted or not.

Reply:
I just put this in, it was from our previous download box html. I'm not sure we even use the status "5" anymore